kernel/private/classes/ezscripttrashpurge.php

```diff
@@ -106,7 +106,7 @@ public function run( $iterationLimit = 100, $sleep = 1 )
         }
         eZUser::setCurrentlyLoggedInUser( $user, $userCreatorID );
 
-        $trashCount = eZContentObjectTrashNode::trashListCount( false );
+        $trashCount = eZContentObjectTrashNode::trashListCount(  array( 'Trashed'  => $trashed ) );
         if ( !$this->quiet )
         {
             $this->cli->output( "Found $trashCount object(s) in trash." );
```

Comment:
This number may now be wrong. If `$trashed` is set, it should be amended to say that it's only counting objects beyond this age.